python/pyarrow/types.pxi

```diff
@@ -1367,6 +1367,28 @@ cdef class Field(_Weakrefable):
 
 
 cdef class Schema(_Weakrefable):
+    """
+    A named collection of types a.k.a schema.
+
```

Comment:
Can you add a reference to `pa.schema` as the function how to create a Schema object manually? (eg with a "See Also" section)